packages/manager/modules/overthebox/src/overthebox/actions/overTheBox-actions.html

```diff
@@ -0,0 +1,80 @@
+<div class="telecom-overTheBox-actions">
+    <div class="telecom-section-content">
+        <oui-message type="error" dismissable data-ng-if="$ctrl.isError">
+            <span data-ng-bind="$ctrl.errorMessage"></span>
+        </oui-message>
+
+        <div class="widget-presentation">
+            <h2
+                class="widget-presentation-title"
+                data-translate="overTheBox_actions_list"
+            ></h2>
+
+            <div
+                class="oui-datagrid-topbar__actions"
+                data-ng-if="!$ctrl.isLoading"
+            >
+                <button
+                    type="button"
+                    class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+                    data-ng-click="$ctrl.getActions()"
+                    title="{{'overTheBox_actions_refresh' | translate}}"
+                >
+                    <span class="fa fa-refresh" aria-hidden="true"></span>
+                </button>
+            </div>
+            <oui-datagrid
+                data-rows="$ctrl.actionIds"
+                data-row-loader="$ctrl.transformItem($row)"
+            >
+                <oui-column
+                    data-title=":: 'overTheBox_actions_name' | translate"
+                    data-property="name"
+                    data-sortable="asc"
+                ></oui-column>
+                <oui-column
+                    data-title="'overTheBox_actions_id' | translate"
+                    data-property="actionId"
+                    data-sortable
+                ></oui-column>
+                <oui-column
+                    data-title="'overTheBox_actions_created_date' | translate"
+                    data-property="createdAt"
+                    data-sortable
+                >
+                    <span
+                        data-ng-bind="$row.createdAt | date:'dd/MM/yyyy'"
+                    ></span>
+                </oui-column>
+                <oui-column
+                    data-title="'overTheBox_actions_updated_date' | translate"
+                    data-property="updatedAt"
+                    data-sortable
+                >
+                    <span
+                        data-ng-bind="$row.updatedAt | date:'dd/MM/yyyy'"
+                    ></span>
+                </oui-column>
+                <oui-column
+                    data-title="'overTheBox_actions_status' | translate"
+                    data-property="status"
+                    data-sortable
+                >
+                    <span
+                        class="oui-status"
+                        data-ng-class="{
+                                'label-success': $row.status === 'done',
```

Comment:
```suggestion
                                'oui-status_success': $row.status === 'done',
```
and same for other status